os/src/Path.scala

```diff
@@ -436,6 +444,33 @@ object Path {
     }
   }
 
+  /**
+   * @return true if Windows OS and path begins with slash or backslash.
+   * Examples:
+   *    rootRelative("/Users")   // true in `Windows`, false elsewhere.
+   *    rootRelative("\\Users")  // true in `Windows`, false elsewhere.
+   *    rootRelative("C:/Users") // false always
+   */
+  def rootRelative[T: PathConvertible](f0: T): Boolean = rootRelative(f0.toString)
+
+  def rootRelative(s: String): Boolean = {
```

Comment:
Do we need this overload, given that the `implicit object StringConvertible extends PathConvertible[String]` already exists? If not we should remove it

Reply:
Combined the two:
```scala
  def rootRelative[T: PathConvertible](f0: T): Boolean = {
    if (platformPrefix.isEmpty) {
      false // non-Windows os
    } else {
      f0.toString.take(1) match {
        case "\\" | "/" => true
        case _ => false
      }
    }
  }
```